srcpkgs/zellij/template

```diff
@@ -0,0 +1,30 @@
+# Template file for 'zellij'
+pkgname=zellij
+version=0.17.0
+revision=1
+archs="~armv*"
```

Comment:
In general, negated `archs` lists are bad. `xbps-src` supports a lot more than the 9 official build architectures. Because `wasmer-vm` explicitly supports `x86_64`, `x86` and `aarch64`, you should do
```sh
archs="x86_64* aarch64* i686"
```
if you can't patch the crate to support more.

Reply:
I'll do this. A dependent crate craps out for armv6/7 saying it's unsupported.

Reply:
Done